openedx_learning/apps/authoring/collections/api.py

```diff
@@ -22,15 +22,15 @@
 
 def create_collection(
     learning_package_id: int,
```

Comment:
```suggestion
def create_collection(
    learning_package_id: int,
    created_by: int | None, 
```

I realized the `created_by` value was missing from the params and in the `.create` below, would you mind adding them in this PR since I needed it when I was working on the REST endpoints.

Reply:
Good catch @yusuf-musleh!
Fixed here: 89f4ceaea18800a1d1093c8f72f76d2428f78ca1